webapp/src/main/webapp/src/app/groups/groups.service.ts

```diff
@@ -51,15 +61,31 @@ export class GroupService {
   private mapGroupFromApi(apiModel): Group {
     let uiModel = new Group();
 
+    uiModel.id = apiModel.id;
     uiModel.schoolYear = apiModel.schoolYear;
     uiModel.name = apiModel.name;
     uiModel.schoolName = apiModel.schoolName;
     uiModel.subject = apiModel.subject;
     uiModel.studentCount = apiModel.studentCount;
+    uiModel.isDeleted = apiModel.isDeleted;
 
     return uiModel;
   }
 
+  private mapGroupToApi(uiModel: Group): any {
+    let apiModel: any = {};
+
+    apiModel.id = uiModel.id;
+    apiModel.schoolYear = uiModel.schoolYear;
+    apiModel.name = uiModel.name;
+    apiModel.schoolName = uiModel.schoolName;
+    apiModel.subject = uiModel.subject;
+    apiModel.studentCount = uiModel.studentCount;
+    apiModel.isDeleted = uiModel.isDeleted;
```

Comment:
I am sending the entire group object back to the API though, in anticipation that at some point the API will be saving all of this data.